applications/app/views/offlinePage.scala.html

```diff
@@ -12,39 +12,12 @@
     @fragments.metaData(offlinePage)
 
     @* get the stylesheets downloading ASAP *@
-    @fragments.stylesheetLinks(None)
-
-    @* polyfill, feature detect etc before we try and use the stylesheets *@
-    @fragments.javaScriptPreFlight(offlinePage)
-
-    @* try and load fonts before we use the stylesheets *@
-    @fragments.fonts()
-
-    @* start trying to use the stylesheets *@
-    @fragments.stylesheetLinksEnable()
+    @fragments.stylesheets(None)
 
+    @* load the app *@
     <script>
-        @* Service worker only caches the bundles, so the offline page can only ask for those. *@
-        var curl = {
```

Comment:
As per the comment, these paths are actually needed by the offline page in dev. I'll raise a PR to restore this